### ISSUE TITLE
Mutations that use Enums 

### ISSUE THREAD

**Issue by steezeburger** [dupski/json-to-graphql-query#2]
Mutations that use Enum values for inputs do not work as it creates the query string with double quotes around the Enum types

GraphQL schema
```
type ProviderHours {
  day: DayOfWeek!
  isOpen: Boolean!
}

enum DayOfWeek {
  MONDAY
  TUESDAY
   ...
}
```

Node
```
let mutation = {
  mutation: {
    updateProviderHours: {
      __args: {
        id: 1,
        input: [{
          day: 'MONDAY',
          isOpen: false
        }]
      },
      hours: {
        day: true,
        isOpen: true
      }
    }
  }
}
mutation = jsonToGraphQLQuery(mutation)
```

the above gives me

```
mutation { updateProviderHours (providerId: 1, input: [{day: "MONDAY", isOpen: false}]) { hours { day isOpen } } }
```

using "MONDAY" in the string.

Is there anyway to not quote these values when they're enumerators?

**Comment by dupski** [comment]
Hmm, I guess we'll need a way to tell the converter that the value is an enum value.

Maybe something like

```
input: [{
   day: 'enum:MONDAY',
   isOpen: false
}]
```

What do you think?

**Comment by dupski** [comment]
Although I guess that would allow user-defined strings to potentially break your queries. Hmm

**Comment by steezeburger** [comment]
I was thinking of ways to implement this in your library earlier today and I think the latter option is the best. I've gotten around this now by looping through my enum values and replacing the double quotes, but having this supported in the library would be fantastic! 

Your library has made creating tests quite a dream so I'd love to see this feature added. I _might_ be able to take a stab at a PR, but not sure when I'll have the time.

**Comment by douglaseggleton** [comment]
🦖https://github.com/dupski/json-to-graphql-query/pull/3

**Comment by dupski** [comment]
Published `v1.2.0` which supports this feature @steezeburger . Cheers @douglaseggleton :)

```
import { jsonToGraphQLQuery, EnumType } from 'json-to-graphql-query';

const query = {
    query: {
        Posts: {
            __args: {
                where: { id: 2 }
                orderBy: 'post_date',
                status: new EnumType('PUBLISHED')
            },
            id: true,
            title: true,
            post_date: true
        }
    }
};
```
